propel/propel/2017-02-14.yaml

```diff
@@ -0,0 +1,8 @@
+title:     SQL injection possible with limit() on MySQL
+link:      https://github.com/propelorm/Propel2/issues/1463
+cve:       ~
+branches:
+    "2.0":
+        time:     2018-02-16 13:38:00
+        versions: ['>=2', '<=3']
```

Comment:
2.x doesn't have any releases at all: this range is empty (alpha versions are not covered)

Reply:
Ah, a contributor on my linked above suggested that `>=2` would include alpha versions. Is that not correct? If not, is there an expression I can use to cover everything in the 2.x series - would `2.*` work?

Reply:
Yeah, `2.0.0` is after `2.0.0-alpha1`, so it is not in the range. You can write in `>=2.0.0-alpha1`.